### PR TITLE
[6.4.x] JBPM-5398 - Transaction issues with Timer catch event

### PR DESCRIPTION
**Pull request by mswiderski**
@sutaakar here is a change we talked about that should fix the problem with kie server tests

### CODE REVIEW

**Comment by mswiderski** [comment]
jenkins retest this

**Comment by mswiderski** [comment]
jenkins retest this

File: jbpm-bpmn2/src/test/resources/BPMN2-BusinessRuleTaskWithDataInput.drl [hunk 1]
```diff
@@ -21,7 +21,7 @@ import org.jbpm.bpmn2.objects.Person;
 rule "Person rule" ruleflow-group "MyRuleFlow"
 no-loop
     when
-        $p : Person()
+        $p : Person(name == null)
```
Comment (psiroky) [inline jbpm-bpmn2/src/test/resources/BPMN2-BusinessRuleTaskWithDataInput.drl]: This should not be needed. The issue seems to be with no-loop + eager evaluation directly in Drools. I believe @mariofusco fixed it for 6.5.x and master.

**Comment by mswiderski** [comment]
won't be included in 6.4.x

**Pull request closed without merging**